projects/igniteui-angular/src/lib/time-picker/time-picker.component.ts

```diff
@@ -542,7 +549,6 @@ export class IgxTimePickerComponent implements
     private _minuteView = [];
     private _ampmView = [];
 
-    private _overlayId: string;
     private _dateFromModel: Date;
     private _destroy$ = new Subject<boolean>();
     private _positionSettings: PositionSettings;
```

Comment:
you do not use this field and may remove it
you also do not need _dialogOverlaySettings field